Formula/kwok.rb

```diff
@@ -0,0 +1,30 @@
+class Kwok < Formula
+  desc "Kubernetes WithOut Kubelet - Simulates thousands of Nodes and Clusters"
+  homepage "https://kwok.sigs.k8s.io"
+  url "https://github.com/kubernetes-sigs/kwok/archive/refs/tags/v0.1.0.tar.gz"
+  sha256 "f8b5d01df09e1b68078c9754bb1922f1e9be3794fd519cd0de615280a0fff84d"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+  depends_on "make" => :build
+  depends_on "docker" => :test
+
+  def install
+    system "make", "build", "VERSION=v#{version}"
+
+    arch = Hardware::CPU.arm? ? "arm64" : "amd64"
+    bin.install "bin/#{OS.kernel_name.downcase}/#{arch}/kwok"
+    bin.install "bin/#{OS.kernel_name.downcase}/#{arch}/kwokctl"
+
+    generate_completions_from_executable(bin/"kwokctl", "completion", shells: [:bash, :zsh, :fish])
```

Comment:
The default should cover all three shells, we only specify when we the binary does not provide all three.

```suggestion
    generate_completions_from_executable(bin/"kwokctl", "completion")
```